rs/cli/src/main.rs

```diff
@@ -354,6 +355,60 @@ async fn async_main() -> Result<(), anyhow::Error> {
                 }
             },
 
+            cli::Commands::ApiBoundaryNodes(api_boundary_nodes) => match &api_boundary_nodes.subcommand {
+                cli::api_boundary_nodes::Commands::Update { nodes, version } => {
+                    runner_instance
+                        .ic_admin
+                        .propose_run(
+                            ic_admin::ProposeCommand::DeployGuestosToSomeApiBoundaryNodes {
+                                nodes: nodes.to_vec(),
+                                version: version.to_string(),
+                            },
+                            ic_admin::ProposeOptions {
+                                title: Some(format!("Update {} API boundary node(s) to {version}", nodes.clone().len())),
+                                summary: Some(format!("Update {} API boundary node(s) to {version}", nodes.clone().len())),
+                                motivation: None,
+                            },
+                            simulate,
+                        )
+                        .await?;
+                    Ok(())
+                }
+                cli::api_boundary_nodes::Commands::Add { nodes, version } => {
+                    runner_instance
+                        .ic_admin
+                        .propose_run(
+                            ic_admin::ProposeCommand::AddApiBoundaryNodes {
+                                nodes: nodes.to_vec(),
+                                version: version.to_string(),
+                            },
+                            ic_admin::ProposeOptions {
+                                title: Some(format!("Add {} API boundary node(s)", nodes.clone().len())),
+                                summary: Some(format!("Add {} API boundary node(s)", nodes.clone().len())),
```

Comment:
Do we want to have a more explanatory message here, or a link to the forum post if you have one?
I believe not many community members will be familiar how these nodes will be created / updated, etc.

Reply:
I didn't quite understand the difference between summary and motivation. How do you guys use it?

Reply:
We should probably merge the two. The idea was that "summary" should include arbitrary description of the change plus a line with `Motivation: <motivation>` and the "motivation" is just the `<motivation>` part. But we're apparently not very consistent in writing good motivations either way. So we can probably merge the two and simplify the code.

Reply:
Ok, I added the motivation as required argument for the two commands. I think it actually does make sense to have summary and motivation. I looked at some previous proposals and it looks good.